recipes/ulid/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "cci.20230615":
```

Comment:
We normally try to avoid these custom cci versions as much as possible, I can see there's an official version release in the github page, maybe that's the one that can be added or maybe the owner of the project would be willing to release a new version?